components/collection/activity/events/Events.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div ref="eventsContainer">
     <div v-if="events.length > 0">
-      <div v-if="desktop" class="columns is-size-7 has-text-grey">
+      <div v-if="desktop" class="columns is-1 is-size-7 has-text-grey">
         <div class="column">
           <span>{{ $t('activity.event.item') }}</span>
         </div>
-        <div class="column">
+        <div class="is-flex is-align-items-center fixed-width-100px">
```

Comment:
did ```column is-1``` didn't work for you?

the ```is-1``` above in line 4 doesn't do anything

Reply:
Thanks, it works.